packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -984,6 +984,11 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       }
     }
 
+    // if (prevProps.reverse !== this.props.reverse) {
+    //   console.log('changing reverse');
+    //   this.setState({ reverse: this.props.reverse });
+    // }
+
```

Comment:
Unintentionally commented out? Or add in a different PR?

Reply:
Ya something from testing when I was implementing combining client state and server state for `ui.table`